File: Xplicity Holidays/Infrastructure/Database/Models/InventoryItem.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.EntityFrameworkCore.Storage.ValueConversion;

namespace Xplicity_Holidays.Infrastructure.Database.Models
{
    public class InventoryItem : BaseEntity
    {
        [Required]
        public string Name { get;set; }

        [Required]
        public string SerialNumber { get; set; }
        
        [Required]
        public DateTime PurchaseDate { get;set; }

        [Required]
        public DateTime ExpiryDate { get; set; }

        public string Comment { get; set; }

        public int InventoryCategoryId { get; set; }

        public InventoryCategory Category { get; set; }

        public int EmployeeId { get; set; }

        public Employee Employee { get; set; }
    }
}

File: Front-End Holidays/FEHol/src/app/models/inventory-item.ts
import { InventoryCategory } from './inventory-category';

export class InventoryItem {
    id: number;
    name: string;
    serialNumber: string;
    price: number;
    purchaseDate: Date;
    expiryDate: Date;
    comment: string;
    category: InventoryCategory;
}